app/assets/stylesheets/nav.scss

```diff
@@ -0,0 +1,24 @@
+.alert-box {
+  text-align: center;
+}
+
+.user-avatar {
+  height: 40px;
+  width: 40px;
+  border: 1px solid #333333;
+  box-shadow: 1px 1px 1px #333333;
+  margin: 0px 5px 3px 8px;
+
+}
+
+nav {
+  box-shadow: 0px 5px 10px #333333;
+}
+
+.admin-info {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.